frontend/public/components/secrets/create-secret.tsx

```diff
@@ -27,14 +29,16 @@ export enum SecretTypeAbstraction {
   webhook = 'webhook',
 }
 
-const secretDisplayType = (abstraction: SecretTypeAbstraction) => {
+const secretDisplayType = (isCreate: boolean, abstraction: SecretTypeAbstraction, t: TFunction) => {
```

Comment:
It's better to move code like this into a block that already has useTranslation or withTranslation rather than passing in a TFunction.

Reply:
But this is before and with this PR a utils function, not a component ?! Hmm, don't know how I could change this without refactoring much more code.

Reply:
I know this isn't new code you added. :) This is just something that's come up in other reviews; I've been dinged for it myself.

It looks like `secretDisplayType` is only called in `SecretFormWrapper.` Could we move this const into that component so we avoid passing the TFunction around?

Reply:
@jerolimov This was the unresolved comment that I was talking about.

Reply:
@rohitkrai03 @rebeccaalpert  Ah thanks for pointing this out. I'm not a big fan of including this inline to an (already long) render function. Esp. where I tried to also keep the diff understandable. In this case I would prefer a "private" local helper function like this one. Also because the other one (one comment below this) has a similar problem but is fine. I think it makes sense that both "translate helpers" work similar here.